rustls/examples/internal/bench_impl.rs

```diff
@@ -252,24 +252,41 @@ fn bench_bulk(
         max_fragment_size,
     ));
 
+    let total_data = options.apply_work_multiplier(if plaintext_size < 8192 {
+        64 * 1024 * 1024
```

Comment:
Nit: potentially extract the `1024 * 1024` factor from the `if`? (A `match` would be slightly more concise.)